util/shlib_wrap.sh.in

```diff
@@ -101,7 +101,7 @@ SunOS|IRIX*)
 	;;
 esac
 
-{- output_off() if $config{ex_libs} !~ /,-rpath,/; ""; -}
+{- output_off() unless grep (/-rpath/, @{$config{LDFLAGS}}); ""; -}
```

Comment:
Just in case for reference, see related https://github.com/openssl/openssl/issues/6500#issuecomment-398714504